colossalai/tensor/colo_tensor.py

```diff
@@ -28,7 +27,7 @@ def __init__(
             pin_memory=False,
             device=None,
             torch_tensor=torch.empty(0),
-            shard_spec: str = None,
+            shard_spec: TensorSpec = None,
```

Comment:
I think the default should be compute pattern as DP?

Reply:
Done